AdyenComponents/Apple Pay/ApplePayComponent.swift

```diff
@@ -116,21 +116,22 @@ public class ApplePayComponent: NSObject, PresentableComponent, PaymentComponent
 
     /// Finalizes ApplePay payment after being processed by payment provider.
     /// - Parameter success: The status of the payment.
+    @available(*, deprecated, message: "This property will no longer be available.")
     public func didFinalize(with success: Bool) {
-        self.success = success
+        finalizeCompletion = {}
```

Comment:
Why not set it to nil? setting it to empty block seems weird

Reply:
it's nullability is used as a flag that `didFinalize` were called do differentiate when `paymentAuthorizationViewControllerDidFinish(_ controller: PKPaymentAuthorizationViewController)` were called before or after `didFinalize`

Reply:
Hmm then do you think this is a clear way to do this? 🤔 
Someone else without this knowledge may just set to `nil`. Creates area for error

Reply:
What do you suggest? Bool flag that mimics it?

Reply:
applied state :)